dxfeed/wrappers/class_utils.py

```diff
@@ -0,0 +1,46 @@
+from typing import Union, Iterable
+import collections
+from warnings import warn
+from datetime import datetime
+import pandas as pd
+
+
+def to_iterable(value: Union[str, Iterable[str]]) -> Iterable[str]:
+    """
+    Function to correctly wrap string into iterable object
+
+    Parameters
+    ----------
+    value: str, iterable
+        String or iterable object
+
+    Returns
+    -------
+    : iterable
+    """
+    if isinstance(value, collections.abc.Iterable):
+        if isinstance(value, str):
+            res = [value]
+        else:
+            res = value
```

Comment:
For what purposes we are using additional variable here?
Why don't just reuse `value` and reassign new value to it in this case we don't need else here 🤔

Reply:
🤦